src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -1017,19 +1017,77 @@ private void buildFullDefenceProceed(EventHistory.EventHistoryBuilder builder, C
             builder.directionsQuestionnaireFiled(dqForProceedingApplicants);
         }
 
-        List<Event> miscText = IntStream.range(0, miscEventText.size())
-            .mapToObj(index ->
-                          Event.builder()
-                              .eventSequence(prepareEventSequence(builder.build()))
-                              .eventCode(MISCELLANEOUS.getCode())
-                              .dateReceived(caseData.getApplicant1ResponseDate())
-                              .eventDetailsText(miscEventText.get(index))
-                              .eventDetails(EventDetails.builder()
-                                                .miscText(miscEventText.get(index))
-                                                .build())
-                              .build())
-            .collect(Collectors.toList());
-        builder.miscellaneous(miscText);
+        if (!featureToggleService.isSDOEnabled()) {
```

Comment:
@sampankumar Is this more what you were thinking off with the flag?
If so, I'll change the tests to match. Notice I am now also sifting between who should & shouldn't proceed to SDO more carefully on line 1036.